test/topology_custom/test_tablets.py

```diff
@@ -100,3 +101,21 @@ async def test_tablet_change_initial_tablets(manager: ManagerClient):
     await cql.run_async("ALTER KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1} AND tablets = {'initial': 2};")
     res = await cql.run_async("SELECT * FROM system_schema.scylla_keyspaces WHERE keyspace_name = 'test'")
     assert res[0].initial_tablets == 2, "initial_tablets not altered"
+
+
+@pytest.mark.asyncio
+async def test_tablet_replication_factor_enough_nodes(manager: ManagerClient):
+    cfg = {'enable_user_defined_functions': False,
+           'experimental_features': ['tablets', 'consistent-topology-changes']}
+    servers = [ await manager.server_add(config=cfg) for _ in range(0, 2) ]
```

Comment:
nitpick: `servers = await manager.servers_add(2)` is shorted and more self-explanatory, and potentially faster.

Reply:
Ah, thanks!